### DISCUSSION TITLE
can't vote on the Polygon proposal

### DISCUSSION THREAD

**Discussion by AvachaBay** [snapshot-labs/snapshot#1367] [Q&A]
nothing happens when select an option and press the button
https://snapshot.org/#/revotu.eth/proposal/0x0a86fd864e9b6fa2577a5b7c8711415882ec125d355e318d2dcf9b205aa6a0c2
![изображение](https://user-images.githubusercontent.com/45499422/147328385-257c03dd-e43f-412f-af51-6e4511796ec5.png)

**Comment by mktcode** [comment]
The UI breaks because there's an issue with loading the results. I don't know what's the root cause of this though.

![image](https://user-images.githubusercontent.com/6792578/147353173-bc8a36f3-f66b-43d4-bfba-e048ea66cfe2.png)

This could probably be handled a bit more gracefully and with at least some feedback in the UI.
